src/phantom.js

```diff
@@ -94,6 +95,12 @@ export default class Phantom {
         this.process.stdout.pipe(new Linerstream()).on('data', data => {
             const message = data.toString('utf8');
             if (message[0] === '>') {
+                //Server end has finished NOOP,
+                //lets allow NOOP again..
+                if (message === '>NOOP') {
+                    this.doing_NOOP = false;
+                    return;
+                }
```

Comment:
Well, my theory is to short circuit as early as possible.  Doing substr will just make more work for the GC, I know it's not a huge amount.  But remember these NOOP's are going 10 times a second, and was the reason I was getting CPU spike's.